### PR TITLE
chore: added tests for organization details component

### LINKED ISSUE #1261: [Tests][Component] Create test for people/widgetViews/OrganizationDetails.tsx
### Context

OrganizationDetails.tsx displays the details and users of an organization add test coverage.

File to be tested
`src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx`

Create a file named
`src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx`

Mock a user organization and pass it as a prop to OrganizationDetails, check the `Organization` interface in /app/src/store/main.ts, and make a mock.

Mock the local storage and add a user, the user pubkey should be the same as the Organization `owner_pubkey` so the user can be the org admin.

Check ```frontend/app/src/bounties/__tests__/bountyTest.spec.ts``` for example

We want to test that the right organization details are displayed 

### Acceptance Criteria

- [ ] Test that the organization name Is visible
- [ ] Test that the Deposit, Withdraw, Edit, View Bounties, and History buttons are visible
- [ ] Test that Edit, and Add User buttons are disabled if the user is not the admin
- [ ] Test that View Bounties is disabled if the organization `bounty_count` is 0 
- [ ] Test that the History, and Withdraw Buttons are disabled if the user is not the admin

You can use this PR as an example `https://github.com/stakwork/sphinx-tribes/pull/1185`

### PR DESCRIPTION
**Pull request by wright-eric**
## Describe your changes
Added test for organization detail component

## Issue ticket number and link
closes #1261 

## Type of change
unit test

## Checklist before requesting a review
- [x] I have performed a self-review of my code

## Added test cases
- [x] Test that the organization name Is visible
- [x] Test that the Deposit, Withdraw, Edit, View Bounties, and History buttons are visible
- [x] Test that Edit, and Add User buttons are disabled if the user is not the admin
- [x] Test that View Bounties is disabled if the organization bounty_count is 0
- [x] Test that the History, and Withdraw Buttons are disabled if the user is not the admin

### CODE REVIEW

**Comment by wright-eric** [comment]
cc @elraphty  @kevkevinpal

File: frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+        />
+      );
+
+      const viewBountiesBtn = getByRole('button', { name: 'View Bounties open_in_new' });
+
+      expect(viewBountiesBtn).toBeDisabled();
+    });
+  });
+
+  it('should disabled hostroy and withdraw button if user is not admin', async () => {
```
Comment (elraphty) [inline frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx:175]: @wright-eric fix the typo error by changing `hostroy` to `history`, so I can merge.

File: frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useRouteMatch: () => ({ url: '', path: '' })
+}));
+
+/**
+ * @jest-environment jsdom
+ */
+describe('UserTickets', () => {
```
Comment (elraphty) [inline frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx:32]: Change `UserTickets` to `OrganizationDetails`

File: frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx (hunk 1 above)
Comment (wright-eric) [reply frontend/app/src/people/widgetViews/__tests__/OrganizationDetails.spec.tsx]: Updated

**Comment by elraphty** [comment]
@wright-eric the typos are not fixed yet `OrganizationDetail` should be `OrganizationDetails`, and `histroy` should be `history`

**Comment by wright-eric** [comment]
> @wright-eric the typos are not fixed yet `OrganizationDetail` should be `OrganizationDetails`, and `histroy` should be `history`

Sorry my bad, fixed it now

**Comment by ecurrencyhodler** [comment]
Paid!

**Pull request merged**